module-2/backend/week_four.md

```diff
@@ -9,15 +9,37 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 
 ### Questions
 
-* What is a cookie?
-* What’s the difference between a session and a cookie?
+* What is a cookie? 
+Cookies are delicious snacks and also a saved id tags stored on a browser's directory or database.
```

Comment:
absolutely delicious 🍪 

they are typically stored on the browser or server side - not necessarily in the database though!